commit 758caafc788fcff9808df0aa12523af38ee11753
Author: Sergey Vlasov <vsu@altlinux.ru>
Date:   Wed Jan 3 18:01:40 2007 +0300

    modprobe: Do not pass cmdline_opts to subordinate modules
    
    Options from the modprobe command line should be passed only to the
    install/remove command for the module specified on the command line,
    and not to modules which the requested module depends on.
    
    Fixes http://bugzilla.kerneltools.org/show_bug.cgi?id=1
    
    Signed-off-by: Sergey Vlasov <vsu@altlinux.ru>
---
 modprobe.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modprobe.c b/modprobe.c
index ea8de74..e704015 100644
--- a/modprobe.c
+++ b/modprobe.c
@@ -830,7 +830,7 @@ static void insmod(struct list_head *list,
 	if (!list_empty(list)) {
 		insmod(list, NOFAIL(strdup("")), NULL, 0, warn,
 		       dry_run, verbose, options, commands, 0, ignore_proc,
-		       strip_vermagic, strip_modversion, cmdline_opts);
+		       strip_vermagic, strip_modversion, "");
 	}
 
 	/* Lock before we look, in case it's initializing. */
@@ -967,7 +967,7 @@ static void rmmod(struct list_head *list,
 	/* Now do things we depend. */
 	if (!list_empty(list))
 		rmmod(list, NULL, 0, warn, dry_run, verbose, commands,
-		      0, 1, cmdline_opts);
+		      0, 1, "");
 	return;
 
 nonexistent_module:

